CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### [0.3.1](https://github.com/Human-Connection/Human-Connection/compare/v0.3.0...v0.3.1) (2020-02-04)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-07T11:01:40Z" title="Friday, February 7th 2020, 12:01:40 pm +01:00">Feb 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,3 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### [0.3.1](https://github.com/Human-Connection/Human-Connection/compare/v0.3.0...v0.3.1) (2020-02-04)
```


Why this change?

Remove this file from the PR and update the changelog once we merged everything in, I would say?